src/modules/tasks.js

```diff
@@ -13,6 +13,9 @@ class Task {
   }
 
   remove() {
+    for (let i = this.number; i < taskList.length; i += 1) {
+      taskList[i].number -= 1;
+    }
     taskList.splice(this, 1);
     refreshList();
```

Comment:
- [ ] This activity requires that you use `Array.filter()` method alongside with the _Clear all completed_ button in removing all completed tasks, which is not implemented in your project. Kindly ensure to improve upon the same.